OCC Retries + Reset (#641)
usnistgov/FIPS201: _FIPS201/requirements.md

The PIN on a PIV Card may need to be reset if the cardholder has
forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an

> invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scaring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries (i.e., PIN or OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.

Reviewer: ```suggestion
invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scaring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation mechanism (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Author: Yep. Adding an 's' to methods and will add it.

The PIN on a PIV Card may need to be reset if the cardholder has
forgotten the PIN or if PIN-based cardholder authentication has been disabled by the usage of an

> invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has developed epidermal scarring or similar, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.

Reviewer 2: ```suggestion
invalid PIN more than the allowed number of retries. OCC may need to be reset if the cardholder has experienced epidermal scarring or similar physical change, resulting in false negative biometric verification decisions, or if OCC has been disabled by exceeding the allowed number of negative biometric verification decisions.  A maximum of 10 consecutive activation retries for each of the activation methods (i.e., PIN and OCC attempts) **SHALL** be permitted unless a lower limit is stipulated by the department or agency.
```

Author: @jimfenton agree. Should it be change**s** (plural)? I can't decide. Should we also add text (footnote?) to discourage re-enrollment for temporary damage?

Reviewer: Search keyword OT-57 from t[he comment on SP 800-73-4](https://csrc.nist.gov/csrc/media/publications/sp/800-73/4/archive/2015-05-29/documents/sp800_73-4_2013_draft_comments_and_dispositions.pdf).  

There, it is stated that OCC reset is seldom done - making an re-enrollment the most likely outcome instead of a reset.  RESET does not enroll fresh biometric references.  Maybe we need to state that re-enrollment is needed in most cases?

The PIV Card's activation methods for OCC may also be 
reset by the card issuer. Before the reset, the issuer 

> **SHALL** perform a biometric verification of the cardholder to the biometric data records in the PIV enrollment record[^occ_reset_characteristic]. If the biometric verification decision is negative or no alternative biometric data records are available, the 

Reviewer 3: ```suggestion
**SHALL** perform a biometric verification of the cardholder to the biometric data records in the PIV enrollment record.[^occ_reset_characteristic] If the biometric verification decision is negative or no alternative biometric data records are available, the 
```
